test/unit/constants.test.ts

```diff
@@ -0,0 +1,37 @@
+describe("BASE URL when", () => {
+	// Mock the process.env
+	beforeEach(() => {
+		jest.resetModules(); // Reset modules to clear cached values
+	});
+
+	describe("NODE_ENV = production", () => {
+		it("should return the production host URL", () => {
+			process.env.NODE_ENV = "production";
+
+			const BASE_URL = require("../../src/utils/constants").BASE_URL;
+
+			expect(BASE_URL).toBe("https://center-be.stamford.dev");
+		});
+	});
+
+	describe("NODE_ENV = beta", () => {
+		it("should return the beta host URL", () => {
+			process.env.NODE_ENV = "beta";
```

Comment:
Ok... ik you have resetModules and all, but XD better to use a mock function for environment variables??

Reply:
lemme try rq

Reply:
as discussed, will mark as "TODO: better to use a mock function for environment variables?"